core/templates/pages/subtopic-viewer-page/subtopic-viewer-page.controller.ts

```diff
@@ -66,6 +68,15 @@ angular.module('oppia').directive('subtopicViewerPage', [
                   subtopicDataObject.getPageContents().getSubtitledHtml());
                 ctrl.subtopicTitle = subtopicDataObject.getSubtopicTitle();
                 PageTitleService.setPageTitle(ctrl.subtopicTitle + ' - Oppia');
+
+                let nextSubtopic =
+                  subtopicDataObject.getNextSubtopic();
```

Comment:
Use parens to clearly indicate that the next line is a continuation: 

```
    let nextSubtopic = (
      subtopicDataObject.getNextSubtopic());
```

Reply:
Done.